design/cluster-api-provider-metal3/allow_disabling_node_disk_cleaning.md

```diff
@@ -82,22 +82,22 @@ metadata:
 spec:
  template:
  spec:
-  disableAutomatedClean: False   #default value. Perform automated cleaning.
+  automatedCleaningMode: None   # default value is Full
```

Comment:
Coming from python looks wrong at first glance :)

This will be a literal string "None", not a yaml `null`  - I guess the enum validation will catch any wrong values, but this might be a bit clearer if it was e.g `automatedCleaningMode: disabled` ?

Reply:
Heh, yeah, that's fair. I complain about Python APIs that are hard to use from golang all the time, so we probably should change that to set a good example. ;-)

I like `Disabled` (the style guide calls for title caps).

Reply:
> the style guide calls for title caps

Do you happen to have a link to that guideline?  I'm wondering if I did the wrong thing using lowercase for the inspect annotation, or if that's just for the structural schema defined for the spec?

Reply:
I think it mostly applies to the API fields, not necessarily annotations. https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants

Reply:
@dhellmann @hardys so you suggest `Enabled/Disabled` instead of `Full/None`, or `Full/Disabled`? I would personally go with `Enabled/Disabled` but okay with other too.

Reply:
I have update the proposal. If you meant `Full/Disabled`, then please let me know and I'll update the proposal & implementation.

Reply:
I think if we're considering some version of a "partial" cleaning implementation, then we want to say "Full" now so there isn't confusion about the difference between "Enabled" and "Partial" later.

Reply:
Thanks, make sense. Updated.